programs/programs/policyengine/tests/test_policy_engine.py

```
@@ -0,0 +1,2548 @@
+        plus age, pregnancy, and TxStateCodeDependency.
+        """
+        result = pe_input(self.screen, [TxChip])
+        household = result["household"]
+
+        # Get units for checking fields
+        people = household["people"]
+        household_unit = household["households"]["household"]
+
+        # Check member-level dependencies
+        head_id = str(self.head.id)
+        spouse_id = str(self.spouse.id)
```

Ruff (0.14.3)

Local variable `spouse_id` is assigned to but never used

Remove assignment to unused variable `spouse_id`

(F841)